frontend/src/styles/colors.sass

```diff
@@ -3,6 +3,7 @@ $white: #FFFFFF
 $white-50: #FFFFFF7F
 $turquoise-100: #61D4F6
 $blue-200: #1C2FBA
+$blue-100: #E0E2F2
```

Comment:
We have this color as inactive-color here, please either use existing or change everywhere